warewulf.spec.in

```diff
@@ -30,12 +29,15 @@ Conflicts: warewulf-ipmi
 
 BuildRequires: make
 
-%if 0%{?sle_version}
+%if 0%{?suse_version}
 BuildRequires: systemd-rpm-macros
 BuildRequires: go
 BuildRequires: firewall-macros
+BuildRequires: firewalld
+BuildRequires: tftp
```

Comment:
I am curious why these need to be installed in order to build the package.

Reply:
After the build the linting checks the ownership of all  files and dirs in the buildroot. As the makefile will install files for firewalld and tftp, these packages are needed to determine the ownership of these directories.